ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,243 @@
+
+| Name                       | Type            | Description                                                                    |
+|----------------------------|-----------------|--------------------------------------------------------------------------------|
+| sender                     | DATA, 20 Bytes  | Address of the sender of this transaction                                      |
+| paymaster                  | DATA, 20 Bytes  | Address of the Paymaster if it is paying for the transaction, `null` otherwise |
+| deployer                   | DATA, 20 Bytes  | Address of the Deployer if it is included in the transaction, `null` otherwise |
+| senderCreationGasUsed      | QUANTITY        | The amount of gas actually used by the sender deployment frame                 |
+| senderValidationGasUsed    | QUANTITY        | The amount of gas actually used by the sender validation frame                 |
+| paymasterValidationGasUsed | QUANTITY        | The amount of gas actually used by the paymaster validation frame              |
+| executionGasUsed           | QUANTITY        | The amount of gas actually used by the execution frame                         |
+| postOpStatus               | QUANTITY        | 1 (success), 0 (failure), or `null` (did not run) status of the `postOp` frame |
+| postOpGasUsed              | QUANTITY        | The amount of gas actually used by the paymaster `postOp` frame                |
+| validationLogs             | ARRAY           | Array of log objects, which this transaction'S VALIDATION FRAME generated.     |
+
+Continued, these fields are shared by all transaction types:
+
+| Name              | Type            | Value                                                                                      |
+|-------------------|-----------------|--------------------------------------------------------------------------------------------|
+| transactionHash   | DATA, 32 Bytes  | Hash of the transaction.                                                                   |
+| transactionIndex  | QUANTITY        | Integer of the transactions index position in the block.                                   |
+| blockHash         | DATA, 32 Bytes  | Hash of the block where this transaction was in.                                           |
+| blockNumber       | QUANTITY        | Block number where this transaction was in.                                                |
+| cumulativeGasUsed | QUANTITY        | The total amount of gas used when this transaction was executed in the block.              |
+| effectiveGasPrice | QUANTITY        | The sum of the base fee and tip paid per unit of gas.                                      |
+| gasUsed           | QUANTITY        | The amount of gas used by this specific transaction alone.                                 |
+| contractAddress   | DATA, 20 Bytes  | The contract address created, if the transaction was a contract creation, otherwise `null` |
```

Comment:
What does contract creation look like in this transaction type?  In "normal" transactions this refers to a contract created by calling address(0) directly, so there can be only one.  AA doesn't have an equivalent, as it's an operation performed by the account implementation.  Are we going to use this for the actual account when deployed?  (i.e. if the transaction had initcode, this address will be identical to `sender`, and if there's no initcode then it's empty)

Reply:
This part of the table is copied over from a documentation for `eth_getTransactionReceipt` without modification, so I did not think about it. 
About having a newly deployed `sender` address also in `contractAddress`, I don't know if it provides any useful information. It is already possible to check if there was a deployment frame by looking at `senderCreationGasUsed` value.
I have no strong feelings about this one way or the other.